index.html

```diff
@@ -337,11 +356,14 @@
 maintenance of pseudonymity are preserved. Note that [=pseudonymity=], on its own, is not
 sufficient to render [=data processing=] [=appropriate=].
 
-A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who, at least in the [=context=] of
-the [=processing=] being discussed, are unable to exercise sufficient
-self-determination for any consent they may provide to be receivable. This includes for
-example children, employees with respect to their employers, people in some situations of
-intellectual or psychological impairment, or refugees.
+A <dfn data-lt="vulnerable">vulnerable person</dfn> is a [=person=] who may be unable
+to exercise sufficient self-determination in the [=context=] of a specific kind of
+[=processing=]. Amongst other things, they should be treated with greater default
+privacy protections and should be considered unable to [=consent=] to the [=processing=].
```

Comment:
There's more to explore here about what a service provider should do as a consequence of their user being unable to consent to anything, but again, we can explore it in a subsequent PR.

Reply:
Do we want to endorse this binary "can consent" / "cannot consent" distinction? For example, kids are more easily coerced or tricked into clicking buttons, but that doesn't make it meaningless to discuss their consent to be messaged by particular other kids in their class.